src/App/Examples/web/customers/index.js

```diff
@@ -0,0 +1,743 @@
+    }
+}
+
+Customers.propTypes = {
+    test: PropTypes.bool
+};
+
+CustomersOverview.propTypes = {
+    setCustomerIndex: PropTypes.func.isRequired
+};
+
+CustomersDetailed.propTypes = {
+    customer: PropTypes.exact({
+        id: PropTypes.string.isRequired,
+        firstName: PropTypes.string.isRequired,
+        lastName: PropTypes.string.isRequired,
+        email: PropTypes.string.isRequired,
+        phone: PropTypes.string.isRequired,
+        location: PropTypes.string.isRequired,
+        status: PropTypes.string.isRequired
+    }).isRequired,
+    customersDetailedTabs: PropTypes.arrayOf(PropTypes.exact({
+        name: PropTypes.string.isRequired,
+        component: PropTypes.object.isRequired
+    })).isRequired
+};
+
+CustomersDetailedOrders.propTypes = {
+    customersDetailedOrdersSteps: PropTypes.arrayOf(PropTypes.shape({
+        title: PropTypes.string.isRequired,
+        ongoing: PropTypes.bool,
+        completed: PropTypes.bool,
+        selected: PropTypes.bool
+    })).isRequired,
+    customersDetailedOrders: PropTypes.arrayOf(PropTypes.number).isRequired
+};
+
+CustomersDetailedInquiries.propTypes = {
+    customersDetailedLatestInquiry: PropTypes.exact({
+        id: PropTypes.string.isRequired,
+        date: PropTypes.string.isRequired,
+        numMessages: PropTypes.number.isRequired,
+        resolved: PropTypes.bool.isRequired,
+        message: PropTypes.string.isRequired
+    }).isRequired,
+    customersDetailedPreviousInquiries: PropTypes.arrayOf(PropTypes.exact({
+        id: PropTypes.string.isRequired,
+        date: PropTypes.string.isRequired,
+        numMessages: PropTypes.number.isRequired,
+        resolved: PropTypes.bool.isRequired,
+        message: PropTypes.string.isRequired
+    })).isRequired
+};
+
+CustomersDetailedCharts.propTypes = {
+    customerIdName: PropTypes.string.isRequired,
+    customerOrders: PropTypes.arrayOf(PropTypes.number).isRequired,
+    customerInquiries: PropTypes.arrayOf(PropTypes.exact({
+        id: PropTypes.string.isRequired,
+        date: PropTypes.string.isRequired,
+        numMessages: PropTypes.number.isRequired,
+        resolved: PropTypes.bool.isRequired,
+        message: PropTypes.string.isRequired
+    })).isRequired
+};
+
+CustomersDetailedDialog.propTypes = {
+    customer: PropTypes.exact({
+        id: PropTypes.string.isRequired,
+        firstName: PropTypes.string.isRequired,
+        lastName: PropTypes.string.isRequired,
+        email: PropTypes.string.isRequired,
+        phone: PropTypes.string.isRequired,
+        location: PropTypes.string.isRequired,
+        status: PropTypes.string.isRequired
+    }).isRequired
+};
+
+export default Customers;
+
+export {
+    CustomersOverview,
+    CustomersDetailed,
+    CustomersDetailedDatePickerGroup,
+    CustomersDetailedOrders,
+    CustomersDetailedInquiries,
+    CustomersDetailedInquiryCard,
+    CustomersDetailedSettings,
+    CustomersDetailedCharts,
+    CustomersDetailedDialog
+};
```

Comment:
This file is quite big? Abstract into smaller files?

Reply:
Good idea :)